include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -806,4 +806,15 @@ def MaterializeDebugInfo :
   let dependentDialects = ["debug::DebugDialect"];
 }
 
+def Lint :
+    Pass<"firrtl-lint", "firrtl::FModuleOp"> {
+  let summary = "An analysis pass to detect static asserts.";
+  let description = [{
+    This pass detects assert ops, whose predicate condition can be statically
+    inferred to be false, that is they will trivially fail any simulation. The
```

Comment:
Should this have a (slightly, for now) more general description?

It seems like presently this already does more than "statically inferred to be false" -- reset signal probably will be false at some point (diagnostic sounds useful) but is it fair to summarize as statically inferred to be false?

Anyway either commit to maintaining this or maybe give you a little wiggle room re:description :+1:.

Reply:
Agree, updated the description.